deployment/pom.xml

```diff
@@ -35,63 +35,23 @@
     <build>
```

Comment:
remove this POM, the deployment directory no longer needs to build anything. I just contains scripts/resources

Reply:
Removed